v2/devdoc/log_context_property_type_if_requirements.md

```diff
@@ -45,7 +53,7 @@ The result is placed in `buffer`, while observing the size of `buffer` to be `bu
 typedef int (*LOG_CONTEXT_PROPERTY_TYPE_COPY)(void* dst_value, void* src_value);
 ```
 
-`LOG_CONTEXT_PROPERTY_TYPE_IMPL(int64_t).copy` copies the value pointed by `src_value` to `dst_value`.
+`LOG_CONTEXT_PROPERTY_TYPE_IF_IMPL(int64_t).copy` copies the value pointed by `src_value` to `dst_value`.
```

Comment:
>int64_t [](http://example.com/codeflow?start=35&length=7)

looks like the description of the interface should be type-less.

Reply:
Fixed, ty.